src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1949,7 +1948,337 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder"], *
         """
         super().unfuse_lora(components=components)
 
+class AuraFlowLoraLoaderMixin(LoraBaseMixin):
+    r"""
+    Load LoRA layers into [`AuraFlowTransformer2DModel`]
+    Specific to [`AuraFlowPipeline`].
+    """
+
+    _lora_loadable_modules = ["transformer"]
+    transformer_name = TRANSFORMER_NAME
+
+    @classmethod
+    @validate_hf_hub_args
+    def lora_state_dict(
```

Comment:
There should be a "Copied from statement ..." here like:
https://github.com/huggingface/diffusers/blob/98930ee131b996c65cbbf48d8af363a98b21492c/src/diffusers/loaders/lora_pipeline.py#L1492